tests/test_appsync.py

```diff
@@ -0,0 +1,46 @@
+import unittest
```

Comment:
Thank you for adding tests!

Reply:
Updated the test to confirm the field resolver message